package.json

```diff
@@ -45,6 +46,7 @@
     "@testing-library/dom": "^8.11.1",
     "@testing-library/jest-dom": "^5.15.0",
     "@testing-library/react": "^12.1.2",
+    "@testing-library/react-hooks": "^12.1.2",
```

Comment:
Per discussion, this ought to be ^8.0.0 and with that, tests pass for me.  w00t!

Reply:
yep